src/classes/ADDR_Account_TDTM.cls

```diff
@@ -47,6 +47,7 @@ public class ADDR_Account_TDTM extends TDTM_Runnable {
     public override DmlWrapper run(List<SObject> listNew, List<SObject> listOld, 
         TDTM_Runnable.Action triggerAction, Schema.DescribeSObjectResult objResult) {
 
+        boolean isBefore = (triggerAction == TDTM_Runnable.Action.BeforeUpdate || triggerAction == TDTM_Runnable.Action.BeforeInsert);
```

Comment:
- Suggestion: `boolean isBefore` -> `Boolean isBeforeContext`
- Do you mind deleting lines 64-70 since they are not used at all?